src/debug/di/arm/cordbregisterset.cpp

```diff
@@ -8,7 +8,6 @@
 
 //
 //*****************************************************************************
-#include "stdafx.h"
```

Comment:
Equivalent files for AMD64 do not have this include, additionally it errors, appears to be missing.